test/fixtures_mysql_test.go

```diff
@@ -3,13 +3,13 @@
 package test_test
 
 import (
-	"context"
-	"github.com/applike/gosoline/pkg/cfg"
+	"database/sql"
 	db_repo "github.com/applike/gosoline/pkg/db-repo"
```

Comment:
please remove all db_repo aliases in the test pkg :+1:

Reply:
✅